Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ruby:2.3.1
+MAINTAINER Kevin Reedy <kreedy@chef.io>
+
+RUN apt-get update && apt-get install -y fakeroot
+
+ADD . /opt/chefdk-build
+
+WORKDIR /opt/chefdk-build/omnibus
+RUN bundle install --without development
+RUN git config --global user.email "nobody@chef.io" && git config --global user.name "Chef"
+RUN bundle exec omnibus build chefdk && bundle exec omnibus clean chefdk
```

Comment:
Why not just install an already built omnibus? You could `curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P chefdk`